subsys/nrf_security/src/ssf_secdom/Kconfig

```diff
@@ -8,4 +8,31 @@ config PSA_SSF_CRYPTO_CLIENT
 	bool
 	prompt "PSA crypto provided through SSF"
 	default y
-	depends on SSF_CLIENT && SSF_PSA_CRYPTO_SERVICE_ENABLED
+	depends on (SSF_CLIENT || SSF_V_2) && SSF_PSA_CRYPTO_SERVICE_ENABLED
+
+if PSA_SSF_CRYPTO_CLIENT
+
+config PSA_SSF_CRYPTO_CLIENT_OUT_BOUNCE_BUFFERS
+	bool "Make sure that all output vectors can be written cache-safe from the crypto engine"
```

Comment:
Should we even have a prompt here? I guess no users want to manually verify that all of their buffers are aligned sufficiently.

Reply:
They could also just take the risk tbh, it really is hard to write into the same dataunit (4 bytes) from the client side as the server side. It allows skipping a copy of the data so can significantly reduce overhead, including the 4K extra heap.

Reply:
I agree with Karsten.

I think it will be quite common to try to save the 4kB and the runtime cost of buffering.

I don't think it would be very difficult for a customer to examine every PSA call in their image.

Reply:
Then I guess the prompt needs an update. What about:

"Support PSA crypto with buffers that are not cache-safe" ?

Reply:
fixed